hack/kind/kind.sh

```diff
@@ -48,15 +48,24 @@ down() {
     kind delete cluster --name apex-dev
 }
 
+cacert() {
```

Comment:
YAS! Thank you!
Adding a note about this to the README would be handy

Reply:
good call